test/web_console/middleware_test.rb

```diff
@@ -50,6 +54,26 @@ def headers
       assert_select '#console'
     end
 
+    test 'render console inside the body tag' do
+      get '/', nil, 'web_console.binding' => binding
+
+      assert_select 'body > #console'
+    end
+
+    test '#acceptable_content_type? is truthy if response format is HTML' do
```

Comment:
This test case seems to be doing the same thing as the above one (`test 'render console inside the body tag'`). We can merge them into one, perhaps.